dll/cpl/timedate/ntpclient.c

```diff
@@ -4,6 +4,7 @@
  * FILE:        dll/cpl/timedate/ntpclient.c
  * PURPOSE:     Queries the NTP server
  * COPYRIGHT:   Copyright 2006 Ged Murphy <gedmurphy@gmail.com>
+ *              Copyright 2017 Doug Lyons
```

Comment:
it's 2018 now.
And I hate to nitpick here, but does 1 line of code really warrant copyright claims?

Reply:
No. It can be removed. My first code changes were much more extensive and involved conversion for time zone offsets. Thomas Faber suggested using System Time and when I made these changes then the code path because very much more simple and I did not remove the copyright line. I agree that it should be removed now. Thanks.